addons/crm/tests/test_crm_lead.py

```diff
@@ -758,3 +758,58 @@ def test_phone_mobile_update(self):
         self.assertEqual(lead.phone, self.test_phone_data[1])
         self.assertEqual(lead.mobile, self.test_phone_data[2])
         self.assertFalse(lead.phone_sanitized)
+
+    @users('user_sales_manager')
+    def test_crm_lead_alias_help(self):
+        crm_team_model = self.env['ir.model']._get_id('crm.team')
+        crm_lead_model = self.env['ir.model']._get_id('crm.lead')
```

Comment:
Seems like we don't use these anymore :)

Usually should be `..._model_id` for raw integer ids, just to make it easier to read